components/dialog/index.css

```diff
@@ -11,377 +11,429 @@ governing permissions and limitations under the License.
-
-
-  --spectrum-dialog-confirm-description-padding: var(--spectrum-spacing-50);
-  --spectrum-dialog-confirm-description-margin: calc(var(--spectrum-spacing-50) * -1);
-  --spectrum-dialog-confirm-footer-padding-top: var(--spectrum-spacing-600);
-  --spectrum-dialog-confirm-gap-size: var(--spectrum-component-pill-edge-to-text-100);
-  --spectrum-dialog-confirm-buttongroup-padding-top: var(--spectrum-spacing-600);
-  --spectrum-dialog-confirm-close-button-size: var(--spectrum-component-height-100);
-  --spectrum-dialog-confirm-close-button-padding: calc(26px - var(--spectrum-component-bottom-to-text-300));
-  --spectrum-dialog-confirm-divider-height: var(--spectrum-spacing-50);
+	--spectrum-standard-dialog-max-width: var(--spectrum-standard-dialog-maximum-width-medium);
+	--spectrum-standard-dialog-min-inline-size: var(--spectrum-standard-dialog-minimum-width);
+	--spectrum-standard-dialog-border-radius: var(--spectrum-corner-radius-extra-large-default);
+
+	/* Fonts */
+	--spectrum-standard-dialog-heading-font-weight: var(--spectrum-title-sans-serif-font-weight);
+	--spectrum-standard-dialog-heading-font-size: var(--spectrum-standard-dialog-title-font-size);
+	--spectrum-standard-dialog-heading-font-style: var(--spectrum-title-sans-serif-font-style);
+	--spectrum-standard-dialog-heading-font-family: var(--spectrum-title-sans-serif-font-family);
+	--spectrum-standard-dialog-description-font-weight: var(--spectrum-body-sans-serif-font-weight);
+	--spectrum-standard-dialog-description-font-size: var(--spectrum-standard-dialog-body-font-size);
+	--spectrum-standard-dialog-description-font-style: var(--spectrum-body-sans-serif-font-style);
+	--spectrum-standard-dialog-description-font-family: var(--spectrum-body-sans-serif-font-family);
+
+	--spectrum-standard-dialog-heading-line-height: var(--spectrum-title-line-height);
+	--spectrum-standard-dialog-description-line-height: var(--spectrum-line-height-200);
+
+	/* Colors */
+	--spectrum-standard-dialog-heading-text-color: var(--spectrum-heading-color);
+	--spectrum-standard-dialog-description-text-color: var(--spectrum-body-color);
+	--spectrum-standard-dialog-background-color: var(--spectrum-background-layer-2-color);
+
+	/* Spacing */
+	--spectrum-standard-dialog-spacing-edge-to-content: var(--spectrum-spacing-500);
+	--spectrum-standard-dialog-spacing-title-to-description: var(--spectrum-spacing-300);
+	--spectrum-standard-dialog-spacing-description-to-footer: var(--spectrum-spacing-500);
+	--spectrum-standard-dialog-spacing-title-to-header-content: var(--spectrum-spacing-400);
+	--spectrum-standard-dialog-spacing-edge-to-header-content-inline-end: var(--spectrum-spacing-400);
+
+	/* Grid */
+	--spectrum-standard-dialog-spacing-grid-padding: var(--spectrum-standard-dialog-spacing-edge-to-content);
+	--spectrum-standard-dialog-spacing-edge-to-close-button: var(--spectrum-spacing-200);
+
+	/* TODO: is there a default token somewhere for this hero height? */
```

Comment:
We previously had a `spectrum-dialog-confirm-hero-height` token that was being used to set the `block-size` of the hero image. The value is 128px for desktop and 160px for mobile. 

When I asked if we had a minimum or default height in mind for the hero image, design said to make it 140px, so I just hardcoded it for now.

Reply:
hmmm probably not. That might have been a note for myself, and the answer is no, a token for this current hero height doesn't exist. There are some older "dialog-confirm" token values for hero height, which I brought up to design, but those aren't correct. They suggested the hardcoded 140px. https://github.com/adobe/spectrum-css/pull/2860/commits/237b0de50ef909ae5a18c8ee224b0f88cd7c463f